test/MusicStore.Spa.Test/project.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "compilationOptions": {
+    "warningsAsErrors": "true"
+  },
+  "dependencies": {
+    "MusicStore.Spa": "",
+    "Microsoft.AspNet.Mvc" : "6.0.0-*",
+    "Microsoft.AspNet.Testing.Logging" :  "1.0.0-*",
+    "xunit.runner.kre": "1.0.0-*"
+  },
+  "commands": {
+    "test": "xunit.runner.kre"
+  },
+  "frameworks": {
+    "aspnet50": { },
+    "aspnetcore50": { }
```

Comment:
Just make sure the tests can run on `aspnetcore50` as well. I think `build.cmd` automatically uses a `coreclr` version of `kre` to run these tests. Right now I think there are some issues with `coreclr`